railties/lib/rails/generators/rails/app/templates/config/initializers/handle_cookies_without_metadata.rb.tt

```diff
@@ -0,0 +1,8 @@
+# Be sure to restart your server when you modify this file.
+
+# Signed and encrypted cookies now have metadata associated with it.
+# If you don't want your existing cookies to be rejected immediately you need to specify a window
+# for how long those cookies should be honored as a Time or ActiveSupport::Duration object.
+
+# Also, setting this value to nil or to a date in the past rejects the existing cookies upfront.
+Rails.application.config.action_dispatch.fail_cookies_without_metadata_after = <%= ActiveSupport::JSON.encode 1.month.from_now %>
```

Comment:
Why did you make this a JSON value here? What's the benefit of this way, as opposed to deferring it? How does this choice mesh with our goal of programmer happiness and readability?

Reply:
Where is `use_cookies_with_metadata` set to true? Seems like we'd ship something that would never actually use metadata? And is it the best approach to split this over 2 options? E.g. is the mental overhead of 2 toggles worth it for end users?